qf_lib/brokers/binance_broker/binance_broker.py

```diff
@@ -171,9 +177,28 @@ def _place_order(self, order: Order) -> Transaction:
             self.logger.info(f'Placing order: failed: {ex}')
             self.logger.error(traceback.format_exc())
 
-    def get_portfolio_value(self) -> float:
-        self.logger.info("Calling get_portfolio_value")
+    def get_portfolio_value(self, time: datetime = None) -> float:
```

Comment:
This won't be used and work in current design (OrderFactory). Changes required.

Reply:
changed it to maintain the same interface